react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -154,6 +182,24 @@ class ParticipantView extends Component {
                     && <Avatar
                         style = { this.props.avatarStyle }
                         uri = { avatar } /> }
+
+                { connectionInterrupted
+
+                    // If the connection was interrupted, draw an overlay on
+                    // top of the video / avatar.
+                    && <View
+                        style = { styles.interruptedOverlay } /> }
+
+                { /* eslint-disable max-len */ }
+
+                { connectionInterrupted && this.props.isLargeView
+                    && <View style = { styles.interruptedTextContainer } >
+                        <Text style = { styles.interruptedText } >
+                            { this.props.t('connection.USER_CONNECTION_INTERRUPTED', { displayName: this.props._participantName }) }
```

Comment:
I couldn't come up with a way to make this line shorter, so I disabled the line length check just for a tiny section. I hope that's alright.

Reply:
how about a render method ? _renderInterruptedLabel ?

Reply:
🤦‍♂️ 🤦‍♂️ 🤦‍♂️  of course!